suite-native/module-passphrase/src/components/ConfirmOnDeviceBottomSheet.tsx

```diff
@@ -0,0 +1,60 @@
+import { BottomSheet, Box, Button, Text } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Translation } from '@suite-native/intl';
+
+import { DeviceTS3Svg } from '../assets/DeviceTS3Svg';
+
+const contentStyle = prepareNativeStyle(utils => ({
+    paddingTop: utils.spacings.small,
+    gap: utils.spacings.large,
+    alignItems: 'center',
+    paddingHorizontal: utils.spacings.small,
+}));
+
+const textContentStyle = prepareNativeStyle(utils => ({
+    gap: utils.spacings.small,
+}));
+
+const buttonStyle = prepareNativeStyle(_ => ({
+    width: '100%',
+}));
+
+type ConfirmOnDeviceBottomSheetProps = {
+    isVisible: boolean;
+    onClose: (isVisible: boolean) => void;
+};
+
+export const ConfirmOnDeviceBottomSheet = ({
+    isVisible,
+    onClose,
+}: ConfirmOnDeviceBottomSheetProps) => {
+    const { applyStyle } = useNativeStyles();
+
+    const handleClose = () => onClose(false);
+
+    return (
+        <BottomSheet isVisible={isVisible} onClose={handleClose} isCloseDisplayed={false}>
+            <Box style={applyStyle(contentStyle)}>
+                <DeviceTS3Svg />
```

Comment:
I suppose other devices should be handled here as well. Do we already have assets for these?

Reply:
We dont have it (asked Sheny already). Good thought!